corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -5,13 +5,15 @@
 
 from elasticsearch.exceptions import RequestError, ConnectionError, NotFoundError, ConflictError
 
-from pillowtop.dao.exceptions import DocumentNotFoundError
 from pillowtop.utils import ensure_matched_revisions, ensure_document_exists
 from pillowtop.exceptions import PillowtopIndexingError
 from pillowtop.logger import pillow_logging
 from .interface import PillowProcessor
 
 
+from corehq.apps.change_feed.document_types import is_deletion
```

Comment:
nit, why so many blank spaces above this?

Reply:
Eh, no real reason. Mainly to mirror what's below it